pkg/uprobetracer/tracer.go

```diff
@@ -93,9 +91,9 @@ type Tracer[Event any] struct {
 	// when users write library names in ebpf section names, it's possible to
 	// find multiple libraries of different archs within the same container,
 	// making this a one-to-many mapping
-	containerPid2Inodes map[uint32][]inodeUUID
+	containerPid2Inodes map[uint32][]uint64
 	// keeps the fd and refCount for each inodeUUID
-	inodeRefCount map[inodeUUID]*inodeKeeper
+	inodeRefCount map[uint64]*inodeKeeper
```

Comment:
I suggest to rename `inodeUUID` to `realInodePtr` everywhere. What do you think?

And the comment before `inodeRefCount` should briefly explain why the real inode ptr is used instead of just the inode number, and why we need to care about getting the underlying file under overlayfs.

Reply:
Do you mean using `realInodePtr` as a new type, and replace all `uint64`?

Reply:
No, uint64 is fine for me. I meant in the comment above and elsewhere in the file.